assets/src/edit-story/components/canvas/layout.js

```diff
@@ -221,7 +225,12 @@ const PageArea = forwardRef(function PageArea(
   ref
 ) {
   return (
-    <PageAreaFullbleedContainer ref={fullbleedRef} data-testid="fullbleed">
+    <PageAreaFullbleedContainer
+      ref={fullbleedRef}
+      data-testid="fullbleed"
+      aria-label={__('Fullbleed', 'web-stories')}
```

Comment:
"Fullbleed" is not really descriptive. Maybe "Fullbleed area" or something? How does VoiceOver announce it?

Reply:
The same applies to e.g. [Display](https://github.com/google/web-stories-wp/blob/7cf07181c4a39995df3552a4f50b86036c2b43af/assets/src/edit-story/components/canvas/displayLayer.js#L54) or [Frames](https://github.com/google/web-stories-wp/blob/7cf07181c4a39995df3552a4f50b86036c2b43af/assets/src/edit-story/components/canvas/framesLayer.js#L93) @barklund WDYT about naming them? Display layer, Frames layer, Fullbleed area?

Reply:
Done in https://github.com/google/web-stories-wp/pull/3643/commits/5e98f9339367d3aa955054fb02904dd47c2e4bae